src/components/MainNavigation/MainNavigation.jsx

```diff
@@ -5,18 +5,22 @@ import BalancePage from '../BalancePage/BalancePage.jsx'
 import MainPage from '../MainPage/MainPage.jsx';
 import ErrorPage from '../ErrorPage/ErrorPage.jsx';
 import NotFoundPage from '../NotFoundPage/NotFoundPage.jsx';
+import TransactionsList from '../TransactionsList/TransactionsList.jsx';
+import TransactionDetails from '../TransactionDetails/TransactionDetails.jsx'
 
 
 import * as Routes from '../../routes';
 
 const MainNavigation = () => {
-  const { HOME, ERROR, NOT_FOUND, BALANCE } = Routes;
+  const { HOME, ERROR, NOT_FOUND, BALANCE, TRANSACTIONS, DETAILS } = Routes;
```

Comment:
on line 12 you can use the following:
```
import { HOME, ERROR, NOT_FOUND, BALANCE, TRANSACTIONS, DETAILS } from '../../routes';
```
also question why you have lines 10 and 11 without any content, looks like at least one of them is redundant

Reply:
Fixed )